DEPENDENCIES.md

```diff
@@ -2787,8 +2788,8 @@ THE SOFTWARE.
 The following text applies to code linked from these dependencies:
 
 * [benchmark 0.1.0]( https://crates.io/crates/benchmark )
-* [glean-core 25.1.0]( https://github.com/mozilla/glean )
-* [glean-ffi 25.1.0]( https://github.com/mozilla/glean )
+* [glean-core 30.0.0]( https://github.com/mozilla/glean )
```

Comment:
Uhm, did we forget to update this?

Reply:
Yup. Working next on making sure we don't in the future.

Reply:
#880